src/NuGet.Services.BasicSearch/Startup.cs

```diff
@@ -246,7 +246,7 @@ private bool InitializeSearcherManager(IndexingConfiguration config, Directory d
                     shouldRetry: e =>
                     {
                         // Retry on any exception (but log it)
-                        _logger.LogError("Startup: An error occurred initializing searcher manager. Going to retry...",
+                        _logger.LogError("Startup: An error occurred initializing searcher manager. Going to retry... Exception: {Exception}",
                             e);
                         _searchTelemetryClient.TrackMetric(SearchTelemetryClient.MetricName.SearchIndexReopenFailed, 1);
```

Comment:
Opened [this issue](https://github.com/NuGet/NuGetGallery/issues/4499) to duplicate exception logs as traces.